File: app/src/main/java/com/neome/api/home/main/msg/MsgChatBlock.java
// neome.ai API - do not change
//

package com.neome.api.home.main.msg;

import com.neome.api.meta.base.Types.EntUserId;
import com.neome.api.nucleus.base.msg.Msg;

@SuppressWarnings("unused")
public class MsgChatBlock extends Msg
{
  public boolean block;

  public EntUserId chatId;
}

File: app/src/main/java/com/neome/api/home/main/msg/MsgChatBlock.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.home.main.msg

import com.neome.api.meta.base.Types.EntUserId
import com.neome.api.nucleus.base.msg.Msg

interface MsgChatBlock : Msg
{
  val block: Boolean
  val chatId: EntUserId
}